benchopt/datasets/simulated.py

```diff
@@ -7,7 +7,7 @@
 
 def make_correlated_data(
         n_samples=100, n_features=50, n_tasks=1, rho=0.6, snr=3,
-        w_true=None, density=0.2, random_state=None):
+        w_true=None, density=0.2, random_state=None, X_density=1):
```

Comment:
I disagree with @agramfort, I think it is okay to put it here as it is more consistent and good practice is to always use keyword for such helper.

```suggestion
        w_true=None, density=0.2, X_density=1, random_state=None):
```